.vscode/settings.json

```diff
@@ -1,36 +0,0 @@
-{
-  "files.exclude": {
-    "**/*.pyc": true,
-    "docs/_build": true,
-    "build": true
-  },
-  "git.ignoreLimitWarning": true,
-
-  "editor.formatOnSave": true,
```

Comment:
why remove this instead of updating / overriding with personal settings?
iiuc, this represents the "workspace settings", which you can update view "user settings".
![Screen Shot 2023-03-09 at 8 39 02 AM](https://user-images.githubusercontent.com/1059168/224092278-79129b70-b580-4311-8051-ddde0a4a9af3.png)



user settings are saved on your computer under your user name and if you want to "save" them for later, you could either just check that in your personal repo, or may be use `Turn on Sync` setting.
![Screen Shot 2023-03-09 at 8 41 34 AM](https://user-images.githubusercontent.com/1059168/224092409-e7cc5f92-9d17-4286-bed3-4d2a2683827c.png)
![Screen Shot 2023-03-09 at 8 41 28 AM](https://user-images.githubusercontent.com/1059168/224092413-1c3c27f7-1522-4732-9b82-60030cc452a5.png)


wdyt?